core/src/main/java/brooklyn/util/internal/ssh/ShellAbstractTool.java

```diff
@@ -296,9 +296,10 @@ public ToolAbstractAsyncExecScript(Map<String,?> props) {
          * The executed command will return immediately, but the output from the script
          * will continue to be written 
          * note that some modes require \$RESULT passed in order to access a variable, whereas most just need $ */
+        @Override
         protected List<String> buildRunScriptCommand() {
             String touchCmd = String.format("touch %s %s %s %s", stdoutPath, stderrPath, exitStatusPath, pidPath);
-            String cmd = String.format("( %s > %s 2> %s < /dev/null ; echo $? > %s ) & disown", scriptPath, stdoutPath, stderrPath, exitStatusPath);
```

Comment:
This command was working fine with the tests, but using `disown` while ending the ssh session doesn't seem right, so I converted it to use `nohup`. Any reason it shouldn't be changed? cc @aledsage